python-sdk/tests_integration/sql/operators/transform/test_transform.py

```diff
@@ -309,6 +492,20 @@ def top_five_animations(input_table: Table) -> str:
 
 
 @pytest.mark.integration
+@pytest.mark.skipif(not DATASET_SUPPORT, reason="Inlets/Outlets will only be added for Airflow >= 2.4")
+def test_inlets_outlets_supported_ds_mssql():
+    """Test Datasets are set as inlets and outlets"""
+    imdb_table = (Table(name="imdb", conn_id="mssql_conn"),)
+    output_table = Table(name="test_name")
+
+    @aql.transform
+    def top_five_animations(input_table: Table) -> str:
+        return "SELECT TOP 5 title, rating FROM {{ input_table }}"
+
+    task = top_five_animations(input_table=imdb_table, output_table=output_table)
+    assert task.operator.outlets == [output_table]
+
+
 @pytest.mark.skipif(DATASET_SUPPORT, reason="Inlets/Outlets will only be added for Airflow >= 2.4")
```

Comment:
should we add `@pytest.mark.integration`?

Reply:
fixed